roles/ci_gen_kustomize_values/molecule/default/converge.yml

```diff
@@ -25,8 +25,20 @@
         (ansible_user_dir,
          'src/github.com/openstack-k8s-operators/architecture') | path_join
       }}
+    artifacts: "{{ cifmw_ci_gen_kustomize_values_basedir }}/artifacts"
   tasks:
-    # We first create a dedicated keypair.
+    - name: Ensure architecture repository is building
+      ansible.builtin.shell:
+        chdir: "{{ cifmw_ci_gen_kustomize_values_architecure_repo }}/{{ item.path }}"
+        cmd: >-
+          {{ ansible_user_dir }}/bin/kustomize build >
+          {{ artifacts}}/vanilla-{{ item.name }}.yaml
+      loop:
+        - name: 'dataplane'
```

Comment:
Are we going to maintain this list?

Reply:
that one here? it's for the testing - unless there's a plan to remove that specific example, we shouldn't have any update... ?